### DISCUSSION TITLE
Clock 2.10

### DISCUSSION THREAD

**Discussion by BlackyHawky** [BlackyHawky/Clock#128] [Announcements]
**Fixes**
- Fix dragging when phone is in landscape mode;
- Fix the color of the seconds hand in the Clock tab (again):
  - Now the color correctly follows the night accent color;

**Improvements**
- Set alarms for a specific date;
- Choose date picker style;
- Add the "Black", "Purple" and "Yellow" themes;
- Reset expired timer with volume and power buttons;
- Add actions to volume buttons for stopwatch:
  - With a single press or a long press, it is possible to start/pause the stopwatch or reset the stopwatch or add an interval or share your time;
- Don't expand the alarm at time change if it is reduced;
- Slight change in the backup file name and the display time of the toast message has been reduced:
  - The new date format for file name is _"yyyy_MM_dd_HH-mm-ss"_;
- Sort accent color names alphabetically;
- Replace the red color with Material error color for the night mode;
- Change the color of icons in settings and in the action bar:
  - The color has been adjusted very slightly to match the text color;
- Optimize and/or update vector icons;
- Update Readme, screenshots and translations:
  - Screenshots (1x1px size) have been added in fastlane folder so that old and now obsolete screenshots can be removed from F-Droid (These files may be deleted after the next update - See https://github.com/BlackyHawky/Clock/discussions/23#discussioncomment-9390212 );
- Improve code;

<hr /><em>This discussion was created from the release <a href='https://github.com/BlackyHawky/Clock/releases/tag/2.10'>Clock 2.10</a>.</em>

**Comment by kale386** [comment]
Photo background, the ability to use photos, instead of just colors. Thank You.
